docs/content/configuration.md

```diff
@@ -102,6 +102,13 @@ these are rarely used for various reasons.
 | `sessionLife`         | `CMD_SESSION_LIFE`          | **`14 * 24 * 60 * 60 * 1000`**, `1209600000` (14 days)                  | Cookie session life time in milliseconds.                                                                                                                      |
 | `sessionSecret`       | `CMD_SESSION_SECRET`        | **`secret`**                                                            | Cookie session secret used to sign the session cookie. If none is set, one will randomly generated on each startup, meaning all your users will be logged out. |
 
+## Guest Notes autolock
+
+| config file           | environment                 | **default** and example value                                           | description                                                                                                                                                    |
```

Comment:
```suggestion
| config file           | environment                 | **default** and example value                                           | description                                                                                                                                                    |
| --------------------- | --------------------------- | ----------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------- |
```

You missed the header / table divider

Reply:
![image](https://user-images.githubusercontent.com/3978662/114430721-04694a00-9bbf-11eb-80f3-aef9e609a997.png)

Reply:
Hope that's fixed now, thanks for pointing it out!

Reply:
looks good to me